lib/acme-client.rb

```diff
@@ -1,3 +1,7 @@
+# This file lives here for backward compatibility only and should be deprecated soon
+# gem named acme-client should be required as
```

Comment:
It should stay forever too, iirc for example bundler autorequire will still just try plain gem name.